### DISCUSSION TITLE
Is vue compatibility no longer working?

### DISCUSSION THREAD

**Discussion by cottrell** [docsifyjs/docsify#1887] [Q&A] [answered]
See here https://github.com/docsifyjs/docsify/blob/develop/docs/vue.md ... it is not rendering there and I've tried it in my own freshly init'd doc. Wondering if there is a known issue here or there is something more complicated that pasting the script src in the index.html now required.

**Comment by s-varenia** [comment, marked as answer]
At a minimum, you need to add the `vueGlobalOptions` key to the Docsify configuration:
```
window.$docsify = {
  name: 'Hello World',
  vueGlobalOptions: {
  },
};
```

**Reply by Koooooo-7** [reply]
Thx @s-varenia  for clarify the inevitable extra config on docsify v4+, we will auto detect the Vue global instance and activate it directly in v5 releases.
